catalog/core/catalog-core-attributeregistry/src/main/java/ddf/catalog/data/impl/AttributeRegistryImpl.java

```diff
@@ -27,27 +27,28 @@
 
 public class AttributeRegistryImpl implements AttributeRegistry {
   private static final Logger LOGGER = LoggerFactory.getLogger(AttributeRegistryImpl.class);
+  private static final String NULL_ATTR_MSG = "The attribute descriptor cannot be null.";
 
   private final ListMultimap<String, AttributeDescriptor> attributeMap =
       Multimaps.synchronizedListMultimap(LinkedListMultimap.create());
 
   @Override
   public void register(final AttributeDescriptor attributeDescriptor) {
-    notNull(attributeDescriptor, "The attribute descriptor cannot be null.");
-    notNull(attributeDescriptor.getName(), "The attribute name cannot be null.");
+    notNull(attributeDescriptor, NULL_ATTR_MSG);
+    notNull(attributeDescriptor.getName(), NULL_ATTR_MSG);
```

Comment:
This could probably still work but this is specifically talking about the descriptor `name`